recipes/easybuild-easyblocks/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: easybuild-easyblocks
+  version: "3.3.1"
+
+source:
+  fn: easybuild-easyblocks-3.3.1.tar.gz
+  url: https://pypi.python.org/packages/7c/03/0ca88b299508689eea650652f5188f9f03d8fc3001670ad37885e71b6e4f/easybuild-easyblocks-3.3.1.tar.gz
+  md5: b53fc64807b56a8bef94b26ae5c5b1a6
```

Comment:
See the example recipe for how the url should be
Use a jinja variable for the version
Also we prefer sha256